transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -193,10 +192,7 @@ public ChannelPipeline addAfter(String baseName, String name, ChannelHandler han
     @Override
     public synchronized ChannelPipeline addAfter(
             EventExecutorGroup group, String baseName, String name, ChannelHandler handler) {
-        AbstractChannelHandlerContext ctx = getContextOrDie(baseName);
-        name = filterName(name, handler);
-        addAfter0(ctx, new DefaultChannelHandlerContext(this, findInvoker(group), name, handler));
-        return this;
+        return addAfter(findInvoker(group), baseName, name, handler);
```

Comment:
Is this change necessary for this PR? If not can we revert this change and submit in a new PR?